src/Symfony/Component/Mailer/Exception/UnsupportedSchemeException.php

```diff
@@ -22,12 +22,20 @@ class UnsupportedSchemeException extends LogicException
     private const SCHEME_TO_PACKAGE_MAP = [
         'gmail' => [
             'class' => Bridge\Google\Transport\GmailTransportFactory::class,
-            'package' => 'symfony/google-mailer',
+            'package' => 'symfony/gmail-mailer',
```

Comment:
The package is called `symfony/google-mailer`

Reply:
Thanks fixed